lib/zmq-kernel.js

```diff
@@ -162,7 +162,7 @@ export default class ZMQKernel extends KernelTransport {
     let startupCode = Config.getJson("startupCode")[displayName];
     if (startupCode) {
       log("KernelManager: Executing startup code:", startupCode);
-      startupCode = `${startupCode} \n`;
+      startupCode += "\n";
```

Comment:
Or wait, do we want a newline or an escaped newline here?

Reply:
I believe a newline is the character we want.